CMakeLists.txt

```diff
@@ -295,6 +297,10 @@ else()
     set(DLT_JSON_LIBRARY "")
 endif()
 
+if (WITH_DLT_LOG_LEVEL_APP_CONFIG)
+    add_compile_definitions(-DDLT_LOG_LEVEL_APP_CONFIG)
```

Comment:
use "add_definitions" to avoid error while enable WITH_LOG_LEVEL_APP_CONFIG

Reply:
thx